Sources/TokamakCore/Tokens/Color.swift

```diff
@@ -61,7 +61,14 @@ public class AnyColorBox: AnyTokenBox {
     }
   }
 
-  public static func == (lhs: AnyColorBox, rhs: AnyColorBox) -> Bool { false }
+  public static func == (lhs: AnyColorBox, rhs: AnyColorBox) -> Bool {
+    lhs.equals(rhs)
+  }
+
+  public func equals(_ other: AnyColorBox) -> Bool {
+    fatalError("implement \(#function) in subclass")
+  }
```

Comment:
Do these two need to be separate for overriding purposes? Please add a doc comment that clarifies it.